xbmc/activity/windows/GUIWindowActivityLog.cpp

```diff
@@ -0,0 +1,334 @@
+/*
+ *      Copyright (C) 2014 Team XBMC
+ *      http://xbmc.org
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "GUIWindowActivityLog.h"
+#include "FileItem.h"
+#include "GUIUserMessages.h"
+#include "URL.h"
+#include "activity/ActivityLog.h"
+#include "filesystem/ActivitiesDirectory.h"
+#include "guilib/GUIWindowManager.h"
+#include "guilib/LocalizeStrings.h"
+#include "guilib/WindowIDs.h"
+#include "input/Key.h"
+#include "settings/Settings.h"
+#include "utils/StringUtils.h"
+#include "utils/URIUtils.h"
+#include "view/ViewStateSettings.h"
+
+#define CONTROL_BUTTON_CLEAR          20
+#define CONTROL_BUTTON_LEVEL          21
+#define CONTROL_BUTTON_LEVEL_ONLY     22
+
+using namespace std;
```

Comment:
at least the theory is that, when you use using namespace std; the compiler will have to consider everything in say string and memory for function lookups and resolutions. On a small project I don't see any noticeable difference but from earlier experiments on removing it from some parts of Kodi there was at least a meassurable difference.

googling it gives varying oppinions so we may put this one in the style category unless someone want to do some perf testing